waltz-ng/client/allocation/components/entity-panel/allocation-entity-panel.html

```diff
@@ -187,4 +159,68 @@ <h4>
         </div>
     </div>
 </div>
+<div class="waep-edit-bar">
+    <div class="row">
+        <div class="col-sm-12">
+            <span ng-if="!$ctrl.editing">
+                <a ng-click="$ctrl.setEditable(true)"
+                   class="clickable">
+                    Edit percentages
+                </a>
+            </span>
+
+            <span ng-if="$ctrl.editing"
+                  class="clickable">
+                <a ng-click="$ctrl.setEditable(false)">
```

Comment:
I would make this a new method: `$ctrl.onCancel()` rather than adding it to `setEditable`.